.github/workflows/gradle.yml

```diff
@@ -3,21 +3,24 @@ name: Java & Go CI
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths-ignore:
+      - 'docs/**'
+      - 'github-actions/**'
+      - 'sechub-doc/**'
+      - 'sechub-website/**'
+      - 'sechub-webui/**'
```

Comment:
Should the sechub-webui really be excluded? It's a deployable Spring Boot app like the others

Reply:
removed sechub-webui from excludes